solidity/scripts/install.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -ex
```

Comment:
We can remove `x` option when we're ready with the script. We won't need this additional output unless we're debugging some problem.